lib/foreman_maintain/concerns/system_helpers.rb

```diff
@@ -213,6 +211,11 @@ def proxy_plugin_name(plugin)
         ruby_prefix(scl) + proxy_plugin_prefix + plugin
       end
 
+      def hammer_plugin_name(plugin)
+        plugin = plugin.tr('_', '-') if debian?
+        [hammer_package, plugin].join(debian? ? '-' : '_')
```

Comment:
Given the commonality between these, it feels like there could be a helper that runs `result.tr('_', '-')`.